site/operational-guide/manual-incremental.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Manual/Incremental Deployment
+menu_order: 20
+---
+This scenario is intended for evaluation, not production use. The
+resulting weave network will survive host reboots without the use of a
+systemd unit as long as Docker is configured to start on boot.
```

Comment:
I read that 2nd sentence as an explanation for why it's "not production use". But it actually sounds like a really good thing. So can we add a sentence for *why* this is not suitable for production use?